src/services/utilServices/SitesService.js

```diff
@@ -29,12 +29,33 @@ const ISOMER_ADMIN_REPOS = [
 ]
 
 class SitesService {
-  constructor({ gitHubService, configYmlService }) {
+  constructor({
+    gitHubService,
+    configYmlService,
+    usersService,
+    isomerAdminsService,
+  }) {
     this.githubService = gitHubService
     this.configYmlService = configYmlService
+    this.usersService = usersService
+    this.isomerAdminsService = isomerAdminsService
   }
 
   async getSites(sessionData) {
+    const isEmailUser = sessionData.getIsEmailUser()
+    let emailAccessibleSites = []
+    const userId = sessionData.getIsomerUserId()
+    const isAdminUser = !!(await this.isomerAdminsService.getByUserId(userId))
+    if (!isAdminUser && isEmailUser) {
+      const user = await this.usersService.findSitesByUserId(userId)
+      if (user) {
+        const { site_members: siteMemberEntries } = user
+        emailAccessibleSites = siteMemberEntries.map((entry) => {
+          const repoData = entry.repo
+          return repoData.name
+        })
+      }
+    }
```

Comment:
this seems abit cluttered - wdyt about combining the nested `if` conditions to make this more clear? alternatively, we could shift this and the conditional check below into helper methods (maybe together with the retrieval?) which is fine too

Reply:
Shifted into a helper method in https://github.com/isomerpages/isomercms-backend/pull/480/commits/88065efa8178db74d582fe792c7242f5780248f7